.github/workflows/system-tests.yml

```diff
@@ -71,6 +71,11 @@ on:
         default: -1
         required: false
         type: number
+      ref:
+        description: "system-tests ref to run the tests on (can be any valid branch, tag or SHA in system-tests repo)"
+        type: string
+        required: false
+        default: 'main'
```

Comment:
Can you remove this line, otherwise, system-tests own CI will never use the good SHA in pull requests.

If I'm not wrong, be leaving it empty, `actions/checkout` behave correctly : 

* main in the context of other repo CI
* head_sha in the context of this repo CI